bgcval2/analysis_timeseries.py

```diff
@@ -88,6 +89,29 @@ def listModelDataFiles(jobID, filekey, datafolder, annual):
     return model_files
 
 
+def list_input_files(files_path, dictionary, paths):
+    """
+    Generate a list of files from a path, which may include 
+    several $PATH values.
+    """
```

Comment:
ideally we should start writing Numpy-style docstrings - that would save us a lot of time when compiling the API documentation

Reply:
Mañana